### ISSUE TITLE
[Google Shopping API] Missing the comparison_link in the response

### ISSUE THREAD

**Issue by hilmanski** [serpapi/public-roadmap#1976]
A customer reported that the response is missing the `comparison_link`. I can confirm this bug. But I'm not sure if this is related to the new layout.

[Intercom](https://app.intercom.com/a/inbox/yzck75jr/inbox/shared/all/conversation/102910000070527?view=List) | [Inspect](https://serpapi.com/searches/66f0cb1b707fd337b0906a71/inspect)

Sidenote:
Currently, we name it `comparisons_link` (with extra s) in the JSON structure overview.

**Comment by hilmanski** [comment]
I'm closing this as the customer can get the current comparison information with our Google Product API.

**Comment by schaferyan** [comment]
@liamcharmer, yes, that is currently the only way to get the `comparison_link`. However, I understand this could require many more search credits if you need to do this for each product. 

I'm reopening this issue so that we can investigate whether we can support this with the Google Shopping API again.
